vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor.Hints
+
+open Microsoft.VisualStudio.FSharp.Editor
+open FSharp.Compiler.CodeAnalysis
+open FSharp.Compiler.Symbols
+open FSharp.Compiler.Text
+open Hints
+
+module InlineParameterNameHints =
+
+    let private getHint (range: range, parameter: FSharpParameter) =
+        {
+            Kind = HintKind.ParameterNameHint
+            Range = range.StartRange
+            Parts = [ TaggedText(TextTag.Text, $"{parameter.DisplayName} = ") ]
```

Comment:
In the inline type hints the name and `:` are separate parts, but here it's just one with the `=`. Is that on purpose? Is there any difference in behavior?

Reply:
@psfinaki How do other editors show them (given, they do)?

Reply:
Yep, actually wanted to discuss this.

So
> How do other editors show them (given, they do)

They don't, I mean not for F#. 

For C#, both type hints and parameter name hints use a colon. My idea was that in F#, to call methods on types, we do this:
```
System.Console.WriteLine(value = "blah")
```
These hints are not implemented yet - but I hope we'll add them soon and that's what I'm future-aligning to.

But we can do any of the following:
1) Use colons everywhere
2) Use equals for whatever parameter names 
2) Use colons for parameter names in functions and equals for parameter names in methods